disnake/widget.py

```diff
@@ -369,11 +379,11 @@ def json_url(self) -> str:
         return f"https://discord.com/api/guilds/{self.id}/widget.json"
 
     @property
-    def invite_url(self) -> str:
+    def invite_url(self) -> Optional[str]:
         """Optional[:class:`str`]: The invite URL for the guild, if available."""
```

Comment:
versionchanged? or was this nullable beforehand?

Reply:
This could return `None` before and was also documented like that, just the return type was incorrect.

Reply:
ah, i see.